### PR TITLE
Fix issue #1179

### PR DESCRIPTION
**Pull request by lydiaaahuang**
**Issue:** 
How to show enum_title selected instead of enum value in headerTemplate?

**Method:**
Initially, when creating an enum, there are enum options. Changed so the enum options are just the enum. Removed options.enum_title, so when creating an enum schema. e.g. the given example with the issue, the user no longer needs to put in enum options. Therefore the enum values are called in the header title. 

**Original:**

```
 var schema = {
   "type": "array",
   "title": "Pets",
   "uniqueItems": false,
   "items": {
     "type": "object",
     "title": "My Pets",
     "headerTemplate": "My Pet is a \"{{self.pet}}\"",
     "properties": {
       "pet": {
         "type": "string",
         "title": "Entity",
         "default": "Contact",
         "enum": [
           "value1",
           "value2",
           "value3"
         ],
         "options": {
           "enum_titles": [
             "Dog",
             "Cat",
             "Elephant"
           ]
         }
       }
     }
   }
 };

```

**Revised:**
```
var schema = {
  "type": "array",
  "title": "Pets",
  "uniqueItems": false,
  "items": {
    "type": "object",
    "title": "My Pets",
    "headerTemplate": "My Pet is a \"{{self.pet}}\"",
    "properties": {
      "pet": {
        "type": "string",
        "title": "Entity",
        "default": "Contact",
        "enum": [
          "Dog",
          "Cat",
          "Elephant"
        ]
      }
    }
  }
  };
```

| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Is backward-compatible?    | ✔️❌
| Tests pass?   | ✔️
| Fixed issues  | comma-separated list of tickets # fixed by the PR, if any
| Updated README/docs?   | ❌
| Added CHANGELOG entry?   | ❌

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -97,6 +97,7 @@
     "webpack-merge": "^4.2.2"
   },
   "dependencies": {
+    "@json-editor/json-editor": "^2.9.0-beta.0",
```
Comment (schmunk42) [inline package.json:100]: Makes no sense to depend on itself. Also there's no package update needed (no changed .lock file)

**Review by schmunk42 — CHANGES_REQUESTED** [review]

**Comment by lydiaaahuang** [comment]
Just added the HTML file I used to test the schema, thought this might be helpful if not you can disregard it.

**Comment by schmunk42** [comment]
We need to think about this change since it could be viewed as a breaking change.
It would be nicer if the user can choose from displaying the value or the title.

CC: @germanbisurgi WDYT? Or is this feasible already?

**Comment by robocoder** [comment]
Not only is this a breaking change, it means the behavior is no longer consistent in`editors/multiselect.js` which has a fallback from enum_titles to enum.

**Comment by germanbisurgi** [comment]
Implemented in #1448

**Pull request closed without merging**